patches/services-network-restricted_cookie_manager.cc.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/services/network/restricted_cookie_manager.cc b/services/network/restricted_cookie_manager.cc
+index 9c1e99cfef255e78942590298d7b7edc1946674e..1a02ecfec9c19973c8d5c977cbe44bd287eacd3e 100644
+--- a/services/network/restricted_cookie_manager.cc
++++ b/services/network/restricted_cookie_manager.cc
+@@ -248,6 +248,7 @@ void RestrictedCookieManager::GetAllForUrl(
+   //                                 removing deprecation warnings.
+   net_options.set_return_excluded_cookies();
+ 
++  BRAVE_GETALLFORURL
```

Comment:
this can't work for network cookies so again this doesn't make sense and is not what I described

Reply:
Based on your comments on Slack @bridiver, it seems you'd like to have `ComputeSameSiteContextForRequest` do the work of storing the top-frame URL into the CookieOptions.
 1. `ComputeSameSiteContextForRequest` isn't called for cookies set via `document.cookie`. Do you imagine it would be called here?
 2. We will be landing support for network cookies in a future PR.

Reply:
as mentioned in slack this was my mistake from searching this page for `BRAVE_SETCANONICALCOOKIE` and not noticing there were two macros with the same name